src/test/java/com/wikia/webdriver/testcases/desktop/discussions/CategoriesTests.java

```diff
@@ -68,33 +69,34 @@
   // fixtures
 
   /**
-   * Runs once before all tests in DESKTOP group, deletes all categories on DISCUSSIONS_1 wiki
+   * Runs once before all tests in DESKTOP group, deletes all categories on qadiscussions (de) wiki
    */
   @BeforeClass(groups = {DESKTOP})
   private void deleteCategoriesDesktop() {
-    deleteCategories(MobileWikis.DISCUSSIONS_1);
+    deleteCategories("qadiscussions", "de");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "qadiscussions" 10 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)